vignettes/ORFikOverview.Rmd

```diff
@@ -87,4 +87,100 @@ There are 7 exons in this group, named 3_* where star is the orf number.
 You will also see that the second orf named 3_2 is twice,
 because that single orf is split on two exons.
 
+# Cage-data for tissue specific 5' utrs
 
+In the previous example we used the reference annotation of the 5' utrs
+from the package GenomicFeatures.
+This is a general reference, which will fail to catch tissue specific orfs.
+Here is an example how to include cage-data, to set new Transcription start sites(TSS).
+
+```r
+library(ORFik)
+library(GenomicFeatures)
+
+# Make BSGenome  example with fasta file and gtf file
+
+# First the gtf file, we will extract the 5' leaders and reassign them
+
+# here we use example data from GenomicFeatures,
+# usually you have your own txdb object, a gtf file etc.
+
+samplefile <- system.file("extdata", "hg19_knownGene_sample.sqlite", package = "GenomicFeatures")
+txdb <- loadDb(samplefile)
+fiveUTRs = fiveUTRsByTranscript(txdb, use.names = TRUE) # <- extract only 5' leaders
+cageData <- "./cageData/heartCage.bed" # location of the cage file
+extension <- 1000 # how much upstream of the original tss
+# can the new tss be ?
+filterValue <- 2 # filter for max cage-peak, how many reads must the max peak have
+# to be valid ?
+cds <- cdsBy(txdb, by = "tx", use.names = TRUE) # if you want to find 
+# upstream open reading frames (uORFs), they can go into the cds.
+# therefor we need the coding sequences.
+changedFiveUTRs <- reassignTSSbyCage(fiveUTRs, cageData, extension,
+                                        filterValue, cds)
+View(changedFiveUTRs)
+
+```
+
+You will now see that most of the transcription start sites have changed.
+For different tissues, you will get different leaders.
+From testing on Fantom5 cage-data, less than 1% of the start sites remain the same
+on average after reassignment by cage.
+
+# Finding tissue specific ORFs
+Using the results from the last example, lets find the tissue specific uORFs from
+the heart cage-data.
+
+```r
+# From the previous two examples we had Hsapiens and changedFiveUTRs.
+# we now assign to null to avoid vignette error.
+changedFiveUTRs <- NULL # get this object from previous example
+Hsapiens <- NULL # get this from the BSGenome example
+
+tx_seqs <- extractTranscriptSeqs(Hsapiens, changedFiveUTRs) # <- Now get only fiveUTRs fastasequences
+
+uORFs <- find_in_frame_ORFs(changedFiveUTRs,tx_seqs)
+View(uORFs)
+```
+The uORFs from this example will most likely contain some uORFs not in the original
+example, also some will be lost. This is, because different tissues transcribes
+different mRNAs.
+
+# Features from ORFs
+
+ORFik contains a big feature set, things like FLOSS, coverage, ORFscore, entropy, 
+translational effiency, inside outside score, distance metrics, etc.
```

Comment:
link to exported features, add "@family features" to them so that they can be grouped and found easily